src/ddmd/blockexit.d

```diff
@@ -36,34 +36,38 @@ import ddmd.visitor;
  */
 enum BE : int
 {
-    BEnone = 0,
-    BEfallthru = 1,
-    BEthrow    = 2,
-    BEreturn   = 4,
-    BEgoto     = 8,
-    BEhalt     = 0x10,
-    BEbreak    = 0x20,
-    BEcontinue = 0x40,
-    BEerrthrow = 0x80,
-    BEany      = (BEfallthru | BEthrow | BEreturn | BEgoto | BEhalt),
+    none      = 0,
+    fallthru  = 1,
+    _throw    = 2,
+    _return   = 4,
+    _goto     = 8,
+    halt      = 0x10,
+    _break    = 0x20,
+    _continue = 0x40,
```

Comment:
https://dlang.org/dstyle.html#naming_keywords (underscore should be appended, not prepended)